exercises/luhn/canonical-data.json

```diff
@@ -21,6 +21,11 @@
       "expected": true
     },
     {
+      "description": "another valid Canadian SIN",
+      "input": "055 444 285",
+      "expected": true
+    },
```

Comment:
In the PR that added this test, the following reasoning was given:

> Part of it was the contrast with 0555-444-285, and part of it was the thought that having a couple of different longer valid numbers might help someone who was struggling to work out how the algorithm needed to be implemented.

I moved it earlier in the suite so that it is more likely encountered by someone who is still trying to work out the algorithm, and updated the language in the test cases that contrast with this one to make that comparison more clear.